Automattic/wp-calypso
# Email: Create Titan Mail checkout flow

#### Changes proposed in this Pull Request

This PR creates the checkout flow for Titan Mail.

#### Testing instructions

* Enable the `titan/phase-2` flag by appending `?flags=titan/phase-2` to the Calypso URL
* Try to add Titan Mail to one of your domains, it should lead to this form where you need to specify the quantity:

<img width="728" alt="titan_step2" src="https://user-images.githubusercontent.com/13062352/103091888-d8b55a80-45f5-11eb-9908-d03703e76395.png">

Verify that Cancel button works as expected by sending you to the /email page, and the Continue button adds the product to the cart.
